### PR TITLE
Feliztoon01.kt

### PR DESCRIPTION
**Pull request by BlackDragon2501**
cdnURL y URL

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

**Review by BlackDragon2501 — COMMENTED** [review]
cdnURL cambio

**Comment by choppeh** [comment]
Add the user ID to get updates from this extension

https://github.com/keiyoushi/extensions/blob/d40d1ff1a55f97b6438be5d53b6738858ca1f23f/index.json#L25481C16-L25481C35

```kt
override val id = 7080800841003944426
```

File: src/ko/blacktoon/src/eu/kanade/tachiyomi/extension/ko/blacktoon/BlackToon.kt [hunk 1]
```diff
@@ -19,16 +19,16 @@ import uy.kohesive.injekt.injectLazy
...
+class Happytoon01 : HttpSource() {
 
-    override val name = "블랙툰"
+    override val name = "해피툰"
 
     override val lang = "ko"
 
     private var currentBaseUrlHost = ""
-    override val baseUrl = "https://blacktoon.me"
+    override val baseUrl = "https://happytoon01.com/"
```
Comment (choppeh) [inline src/ko/blacktoon/src/eu/kanade/tachiyomi/extension/ko/blacktoon/BlackToon.kt:29]: Remove the slash

```suggestion
    override val baseUrl = "https://happytoon01.com"
```

File: src/ko/blacktoon/src/eu/kanade/tachiyomi/extension/ko/blacktoon/BlackToon.kt [hunk 2]
```diff
@@ -19,16 +19,16 @@ import uy.kohesive.injekt.injectLazy
 import kotlin.math.min
 import kotlin.random.Random
 
-class BlackToon : HttpSource() {
+class Happytoon01 : HttpSource() {
```
Comment (choppeh) [inline src/ko/blacktoon/src/eu/kanade/tachiyomi/extension/ko/blacktoon/BlackToon.kt:22]: No need to add the number to the class name.

**Pull request closed without merging**